hawkmoth/util/doccompat.py

```diff
@@ -17,11 +17,13 @@
 # facilitate any kind of migration to Hawkmoth. The compat code could be turned
 # into a fairly simple plugin architecture, with some basic compat builtins, and
 # the users could still extend the compat features to fit their specific needs.
```

Comment:
Hmm, I was comparing with the RFC notes and I noticed this `FIXME` is still here. Is it not obsolete with this series?

Reply:
Not yet. I have the implementation pretty much ready locally, but I had some issues testing it.

Reply:
Alright. looking forward to it then ;)